src/main/java/org/spongepowered/api/entity/living/player/tab/TabList.java

```diff
@@ -65,6 +65,14 @@
     void setFooter(Text footer);
 
     /**
+     * Sets this list's header and footer.
+     *
+     * @param header The new header
+     * @param footer The new footer
+     */
+    void setHeaderAndFooter(Text header, Text footer);
```

Comment:
Calling `setHeader` and `setFooter` in a default method like this would be exactly what this method is trying **to avoid**.